### PR TITLE
Port Aberrant Organs: The flustercluckening

### PR DESCRIPTION
**Pull request by Michiyamenotehifunana**
## About The Pull Request
This PR ports Aberrant Organs from upstream. See the wiki for actual information. https://wiki.cev-eris.com/Aberrant_Organs_ErisEn

In a nutshell, the new Bio-Engineer role creates custom organs for people, that can do stuff like buffing their stats in response to damage; or make organ mods to install into existing ones.

~~This PR does not include any map changes, which will be REQUIRED for the bio-engineer role to really exist.~~
Basic bioengineering lab has been mapped in.

## Why It's Good For The Game
Actual content, new features, new jobs. Lots of fun. Also pre-requisite for Erismed4.

## Changelog
```changelog
add: Added the Bio-engineer role! Makes custom aberrant organs and organ mods.
add: Added aberrant organs! Special organs that do certain things in response to stimuli.
add: Added organ mods! Simple stuff to tack onto your existing organs to make them better.
add; The ship has been refitted with a brand new bioengineering lab!
```

### CODE REVIEW

File: code/modules/client/preference_setup/loadout/lists/eyegear.dm [hunk 1]
```diff
@@ -32,7 +32,7 @@
 /datum/gear/eyes/sciencegoggles
 	display_name = "Science Goggles"
 	path = /obj/item/clothing/glasses/powered/science
-	allowed_roles = list(JOBS_SCIENCE)
+	allowed_roles = list(JOBS_SCIENCE,"Moebius Bio-Engineer")
```
Comment (Shadow-Quill) [inline code/modules/client/preference_setup/loadout/lists/eyegear.dm:35]: ?

**Review by Shadow-Quill — DISMISSED** [review]

**Comment by RadiantFlash7** [comment]
If you don’t mind waiting a week for it to be mapped in, I can map your bioengineer job in. I wanted to make some adjustments to medical anyhow, so…

**Comment by Michiyamenotehifunana** [comment]
StrongDMM_dyXfHXxn1i
dreamseeker_fm8CYX0CZA
dreamseeker_9LstxRxc5w

Bioengineering lab mapped in, medbay upper corridor has been de-mainted, all the loot has been nudged into the maint corridors right besides it.

**Comment by Shadow-Quill** [comment]
Checks are failing.
```
[10:48:59] Runtime in organ_fabricator.dm,43: Cannot read null.known_designs
  proc name: Initialize (/obj/machinery/autolathe/organ_fabricator/Initialize)
  src: Organ Fabricator (/obj/machinery/autolathe/organ_fabricator/loaded)
  src.loc: the floor (100,119,4) (/turf/simulated/floor/tiled/white/brown_platform)
  call stack:
  Organ Fabricator (/obj/machinery/autolathe/organ_fabricator/loaded): Initialize(1)
  Atoms (/datum/controller/subsystem/atoms): InitAtom(Organ Fabricator (/obj/machinery/autolathe/organ_fabricator/loaded), /list (/list))
  Atoms (/datum/controller/subsystem/atoms): InitializeAtoms(null)
  Atoms (/datum/controller/subsystem/atoms): Initialize(389293)
  Master (/datum/controller/master): Initialize(10, 0)

**Review by Shadow-Quill — APPROVED** [review]

**Pull request merged**